manifests/charts/istio-control/istio-discovery/files/waypoint.yaml

```diff
@@ -304,7 +304,7 @@ metadata:
     name: "{{.Name}}"
     uid: "{{.UID}}"
 spec:
-  ipFamilyPolicy: PreferDualStack
```

Comment:
> We want IPv6 addresses allocated automatically if a cluster supports dualstack.

That is what `PreferDualStack` already does - why make this configurable, then, versus simply hardcoding to `PreferDualStack` for all gateways?

I might be missing your use case tho.

Reply:
I can make it PreferDualStack without the annotation but is supporting `RequireDualStack` worth it?

Reply:
> I can make it PreferDualStack without the annotation but is supporting `RequireDualStack` worth it?

I'd rather do that for now, not add yet another a knob/annotation, and wait for someone  to make a good case for `RequireDualStack` in gateways specifically.

Reply:
Works for me, can you add the `no-release-note label` please?